evm_arithmetization/src/fixed_recursive_verifier.rs

```diff
@@ -434,6 +438,56 @@ where
     }
 }
 
+/// Data for the block wrapper circuit, which is used to generate a wrapped
+/// final block proof and obfuscate the remaining private elements of a chain.
+#[derive(Eq, PartialEq, Debug)]
+pub struct BlockWrapperCircuitData<F, C, const D: usize>
+where
+    F: RichField + Extendable<D>,
+    C: GenericConfig<D, F = F>,
+{
+    pub circuit: CircuitData<F, C, D>,
+    parent_block_proof: ProofWithPublicInputsTarget<D>,
+    public_values: FinalPublicValuesTarget,
+    local_vk: VerifierCircuitTarget,
```

Comment:
I didn't like calling it cyclic as there is no cyclic verification (and I think we should have done the same with rook circuit), but I can change it if you prefer cyclic for consistency

Reply:
I'm in favor of calling it `cyclic_vk` since it matches the format of the other `cyclic_vk`s and is at the same location, but I wouldn't mind changing `RootCircuit` instead.